core/src/main/java/jenkins/model/DownloadSettings.java

```diff
@@ -58,32 +58,10 @@
         return GlobalConfiguration.all().getInstance(DownloadSettings.class);
     }
 
-    private boolean useBrowser = false;
-    
-    public boolean isUseBrowser() {
-        return useBrowser;
-    }
-
-    public void setUseBrowser(boolean useBrowser) {
-        this.useBrowser = useBrowser;
-        save();
-    }
-
     @Override public @Nonnull GlobalConfigurationCategory getCategory() {
```

Comment:
There are no longer any actual fields here, so I think `DownloadSettings` could be deleted altogether, and `DailyCheck` moved into its own top-level class or whatever.

Failing that, at least empty out `config.groovy` and delete `help-useBrowser.html`.